File: main/maximum_subarray/javascript/OofNN_bruteForce.js
var maxSubArray = function(nums) {
    let maxSum, currentSum;
    for(let i = 0; i < nums.length-1 ; i++){
        if(i == 0) maxSum = nums[i];
        currentSum = nums[i]
        for(let j = i+1; j<nums.length-1; j++){
            currentSum += nums[j];
            if(currentSum > maxSum)maxSum = currentSum     
        }
    }
     return maxSum;
}
 

File: main/maximum_subarray/python/OofNN_bruteForce.py
def maximum_sub_array (nums):
    for i in range(len(nums)):
        if i==0: max_sum = current_sum = nums[0]
        current_sum = nums[i]
        for j in range (len(nums)):
            current_sum =current_sum + nums[j]
            if current_sum>max_sum: max_sum=current_sum
    return max_sum

